tools/sklearn/model_validation.xml

```diff
@@ -67,7 +69,13 @@ y = read_columns(
 y=y.ravel()
 
 options = params["model_validation_functions"]["options"]
-options['cv'] = get_cv( options['cv'] )
+splitter, groups = get_cv(options.pop('cv_selector'))
+if groups is None:
+    options['cv'] = splitter
+elif groups == "":
+    options['cv'] = list( splitter.split(X, y, groups=None) )
```

Comment:
I see two empty spaces before and after `list`. But, in line `72` the spaces are not present. In general, we should stick to a similar standard.